jina/parsers/helper.py

```diff
@@ -55,6 +55,28 @@ def __call__(self, parser, args, values, option_string=None):
         setattr(args, self.dest, d)
 
 
+class ListofDictsAction(argparse.Action):
+    """argparse action to handle input and convert to List of Dictionaries
```

Comment:
what does the input need to look like for this to work?

Reply:
I will work on Documentation later